eNMS/templates/error.html

```diff
@@ -12,8 +12,14 @@
     <div class="col-middle">
       <h1 class="error-number">{{ error }}</h1>
       {% if error == 403 %}
-      <h2>Access Denied</h2>
-      <p>Full authentication is required to access this resource.</p>
+        <h2>Access Denied</h2>
+        {% if not current_user.is_authenticated %}
+          <p>Full authentication is required to access this resource.</p>
+          <p><a href="{{ url_for('blueprint.route', page='login') }}"><button role="button" class="btn btn-primary">Return to Login</button></a></p>
+        {% else %}
+          <p>An error has occurred.  This can include RBAC-related errors.</p>
```

Comment:
- `An error has occurred` isn't consistent with the other errors, where we don't say that
- `RBAC` is not an acronym I expect users to be familiar with. I'd prefer to say something about permissions, or access rights
- If you're logged in and you get a 403 error, what can it include that isn't RBAC related ? I can't think of anything, but my point is that either we know for sure that's a RBAC problem, then we can say so, or we don't know for sure, and we shouldn't say anything ? This kind of formulation "this can include" is just confusing to me.